### DISCUSSION TITLE
Binding for libsql_dart

### DISCUSSION THREAD

**Discussion by dikatok** [simolus3/drift#3188] [Ideas]
Hi, I would love to develop a `drift` binding for https://pub.dev/packages/libsql_dart

I noticed you have implemented the `hrana` protocol with its binding, would it be okay?

**Comment by simolus3** [comment]
Oh nice, yeah that would definitely be a great addition to the ecosystem! As you've seen in `drift_hrana`, drift has a stable API to make it talk to lower-level database packages. Publishing a `drift_libsql` (or something like that) in a separate package should work well, and I'm happy to help if you have any questions.
Once you've published a version, I can link it from the documentation site about libsql, since I think it's even more useful than my packages which don't offer any synchronization mechanisms.

**Comment by dikatok** [comment]
@simolus3 there you go https://pub.dev/packages/drift_libsql

**Reply by simolus3** [reply]
Looks great! I've added it to the documentation website in d3fa373c7f0260431d3ab04c5a91a8577b0997cc, it will be published with the next drift release.

**Reply by dikatok** [reply]
thanks, appreciate it
